zerodark-theme.el

```diff
@@ -121,6 +123,9 @@
                                                (propertize "RO " 'face 'bold))
                                            "")))
 
+(defvar zerodark-modeline-evil '(:eval (if evil-mode
+                                           (propertize (powerline-evil-tag) 'face 'bold) "")))
```

Comment:
Couldn't `evil-mode` be on without `powerline-evil` being loaded? If so, this will fail, right?